cli/bench/main.rs

```diff
@@ -308,7 +313,7 @@ fn run_strace_benchmarks(
         deno_exe.to_str().unwrap(),
       ])
       .args(args.iter())
-      .stdout(Stdio::null())
+      .stdout(Stdio::inherit())
```

Comment:
Should we revert this one?

Reply:
🤷‍♂️. It's useful for debugging. Can remove though

Reply:
It's okay for me